R/adjust-equivocal-zone.R

```diff
@@ -128,16 +128,15 @@ required_pkgs.equivocal_zone <- function(x, ...) {
 
 #' @export
 tunable.equivocal_zone <- function(x, ...) {
-  tibble::new_tibble(list(
```

Comment:
I have no arguments here, but I'm curious why you made that change. Is it for readability (since these aren't likely to be called a large number of times)?

Reply:
Thanks for calling that out. I'd like to revisit that and return to using `new_tibble()`, though there was a vector recycling issue that resulted in an interval vctrs error later on; I think we'll just need to do some manual `rep()`ping.